components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/model/UserSession.java

```diff
@@ -90,4 +94,20 @@ public String getSessionId() {
     public void setSessionId(String sessionId) {
         this.sessionId = sessionId;
     }
+
+    public String getUserId() {
+        return userId;
+    }
+
+    public void setUserId(String userId) {
+        this.userId = userId;
+    }
+
+    public Long getCreationTime() {
+        return creationTime;
+    }
+
+    public void setCreationTime(Long creationTime) {
+        this.creationTime = creationTime;
+    }
```

Comment:
Formatting error. We need to add a new line after the method signature. Shall we fix this?

Reply:
Source code re-formatted as per suggested.